.devops/deploy-pipelines.yml

```
@@ -0,0 +1,165 @@
+            displayName: 'Skip release bundle'
+
+
+  # Prepare Artifact
+  - stage: Deploy_staging
+    dependsOn:
+      - Release
+    jobs:
+      - job: 'prepare_artifact_and_deploy'
+        steps:
+          # Build application
+          - template: azure-templates/setup-project.yml
+            parameters:
+              # On the assumption that this stage is executed only when Relase stage is,
+              #  with this parameter we set the reference the deploy script must pull changes from.
+              # The branch/tag name is calculated from the source branch
+              #  ex: Build.SourceBranch=refs/heads/master --> master
+              #  ex: Build.SourceBranch=refs/tags/v1.2.3-RELEASE --> v1.2.3-RELEASE
+              gitReference: ${{ replace(replace(variables['Build.SourceBranch'], 'refs/tags/', ''), 'refs/heads/', '') }}
+          - script: |
+              yarn predeploy
+            displayName: 'Build'
+
+          # Install functions extensions
+          - task: DotNetCoreCLI@2
+            inputs:
+              command: "build"
+              arguments: "-o bin"   
+          # Copy application to  
+          - task: CopyFiles@2
+            inputs:
+              SourceFolder: '$(System.DefaultWorkingDirectory)'
+              TargetFolder: '$(Build.ArtifactStagingDirectory)'
+              Contents: |
+                **/*
+                !.git/**/*
+                !**/*.js.map
+                !**/*.ts
+                !.vscode/**/*
+                !azure-templates/**/*
+                !azure-pipelines.yml
+                !.prettierrc
+                !.gitignore
+                !README.md
+                !jest.config.js
+                !local.settings.json
+                !test
+                !tsconfig.json
+            displayName: 'Copy deploy files'
+            
+          - task: AzureFunctionApp@1  
+            inputs:
+              azureSubscription: '$(PRODUCTION_AZURE_SUBSCRIPTION)'
+              resourceGroupName: '$(PRODUCTION_RESOURCE_GROUP_NAME)'
+              appType: 'functionApp'
+              appName: '$(PRODUCTION_APP_NAME)'
+              package: '$(Build.ArtifactStagingDirectory)/'
+              deploymentMethod: 'auto'
+              deployToSlotOrASE: true
+              slotName: 'staging'
+            displayName: Deploy to staging slot
+
+  # Check that the staging instance is healthy
+  - stage: Healthcheck
+    dependsOn:
+      - Deploy_staging
+    jobs:
+      - job: 'do_healthcheck'       
+        steps:  
+          - template: templates/rest-healthcheck/template.yaml@pagopaCommons 
+            parameters:
+              azureSubscription: '$(PRODUCTION_AZURE_SUBSCRIPTION)'
+              appName: '$(PRODUCTION_APP_NAME)'
+              endpoint: 'https://$(PRODUCTION_APP_NAME)-staging.azurewebsites.net/api/v1/info'
```

```suggestion
              endpoint: 'https://$(PRODUCTION_APP_NAME)-staging.azurewebsites.net/info'
```
https://github.com/pagopa/io-functions-assets/blob/master/Info/index.ts#L13

ps: this why is more useful have a standard endpoint